app/routes.py

```diff
@@ -16,6 +18,26 @@ def checker():
     return make_response(jsonify({'success': True, 'message': 'OK', 'version': APP_VERSION}))
 
 
+@app.route('/checker/ready', methods=['GET'])
+def readiness():
+    sphinx_ok_string = 'READY\n'
+    try:
+        with open(SPHINX_BACKEND_READY, 'r', encoding='utf-8') as fd:
+            content = fd.read()
+    except IOError:
+        content = ""
+
+    if content != sphinx_ok_string:
+        logger.error(
+            'Incomprehensible sphinx backend %s answer: %s. '
+            'sphinx search service is probably not ready yet.',
+            SPHINX_BACKEND_READY,
+            content,
+        )
```

Comment:
The log here is not necessary as the `abort()` will be already logged here https://github.com/geoadmin/service-search-wsgi/blob/a8055ecb2f59d0ec7af8d4fa7e62d332a7eedfcc/app/__init__.py#L93